<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendsitem</class>
 <widget class="QWidget" name="friendsitem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border:0px;</string>
  </property>
  <widget class="QLabel" name="friend_chatname_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>25</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="friendhead_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
